app/models/quote.rb

```diff
@@ -9,7 +9,7 @@ class Quote < ActiveRecord::Base
   serialize :text
   validate :reporter_and_user_differ
 
-  encrypts :text
+  has_encrypted :text
```

Comment:
Is this from a blackbox upgrade?

Reply:
Rails 7 krijgt active record encryptie dus sommige lockbox methodes worden rails interne methodes. Lockbox heeft gekozen rails defaults uit de weg te gaan.

Reply:
(Engels blijft de standaard voor me, vergeet steeds om het in NL te doen :facepalm:)

Gaat deze wijziging vanzelf goed? Zelfde encryptieschema e.d.?

Reply:
> (Engels blijft de standaard voor me, vergeet steeds om het in NL te doen 🤦)
> 
> Gaat deze wijziging vanzelf goed? Zelfde encryptieschema e.d.?

Ik lees beiden, geen probleem 

Ja gaat gewoon goed.